TESTING.rst

```diff
@@ -1442,6 +1442,88 @@ You can also run complete k8s tests with
 This will create cluster, build images, deploy airflow run tests and finally delete clusters as single
 command. It is the way it is run in our CI, you can also run such complete tests in parallel.
 
+Manually testing release candidate packages
+===========================================
+
+Breeze can be used to test new release candidates of packages - both Airflow and providers. You can easily
+turn the CI image of Breeze to install and start airflow for both Airflow and provider packages - both,
+packages that are built from sources and packages that are downloaded from PyPI when they are released
+there as release candidates.
+
+The way to test it is rather straightforward:
+
+1) Make sure that the packages - both ``airflow`` and ``providers`` are placed in the ``dist`` folder
+   of your Airflow source tree. You can either build them there or download from PyPI (see the next chapter)
+
+2) You can run ```breeze shell`` or ``breeze start-airflow`` commands with adding the following flags -
+   ``--mount-sources remove`` and ``--use-packages-from-dist``. The first one removes the ``airflow``
+   source tree from the container when starting it, the second one installs ``airflow`` and ``provider``
+   packages from the ``dist`` folder when entering breeze.
+
+Testing pre-release packages
+----------------------------
+
+There are two ways how you can get airflow packages in ``dist`` folder - by building them from sources or
+downloading them from PyPI.
+
+.. note ::
+
+    Make sure you run ``rm dist/*`` before you start building packages or downloading them from PyPI because
+    the packages built there already are not removed manually.
+
+In order to build apache-airflow from sources, you need to run the following command:
+
+.. code-block:: bash
+
+    breeze release-managment prepare-airflow-package
```

Comment:
Typo: should be release-management

Reply:
Right. Fixed. Also added one more example and updated some wording.